css-contain-2/Overview.bs

```diff
@@ -1681,6 +1681,10 @@ Suppressing An Element's Contents Entirely: the 'content-visibility' property {#
 			content-visibility/content-visibility-with-top-layer-005.html
 			content-visibility/content-visibility-with-top-layer-006.html
 			</wpt>
+
+		* The element or one of its ancestors participates in an ongoing {{ViewTransition}},
```

Comment:
This isn't quite the right text for the resolution.

The resolution states that the element is relevant to the user if it has a (flat tree) **descendant** that is participating in an ongoing ViewTransition.

Reply:
Right, will reword.